module-domain/src/main/kotlin/site/yourevents/guest/service/GuestService.kt

```diff
@@ -81,6 +82,11 @@ class GuestService(
         return guestPersistencePort.findNotAttendGuestsByInvitation(invitation)
     }
 
+    override fun getInvitationAttendance(memberId: UUID, invitationId: UUID): Boolean? {
+        val guest = guestPersistencePort.findByMemberAndInvitation(memberId, invitationId)
+        return guest
+    }
```

Comment:
참석 여부를 반환하는데 변수명은 `guest`인 이유가 있을까요?
그리고 단일 표현식으로 표현이 가능할 것 같아보입니다..!
```suggestion
    override fun getInvitationAttendance(memberId: UUID, invitationId: UUID) = 
        guestPersistencePort.findByMemberAndInvitation(memberId, invitationId)
```

Reply:
아 넵!! 알겠습니다!! Boolean으로 수정하였기에 단일 표현식이 가능하군요!! 수정하였습니다.